kvmagent/kvmagent/plugins/network_plugin.py

```
@@ -381,6 +381,16 @@ def del_novlan_bridge(cmd):
         cmd.bridgeName, cmd.physicalInterfaceName))
 
 
+def update_global_variables_for_net_config():
+    if not os.path.exists('/usr/local/bin/zsha2'):
+        return
+
+    r, o = bash_ro("/usr/local/bin/zsha2 show-config")
```

Ruff (0.8.2)

`bash_ro` may be undefined, or defined from star imports

(F405)